src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -41,12 +42,16 @@
 class AreaEncoder(json.JSONEncoder):
     """Convert the Area class hierarchy to json dict."""
     def default(self, o):
-        if isinstance(o, AreaBase):
+        # Leaf classes are Areas too, therefore the Area/AreaBase classes need to be handled
+        # separately.
+        if type(o) in [Area, AreaBase]:
             return self._encode_area(o)
         if isinstance(o, Leaf):
             return self._encode_leaf(o)
         if isinstance(o, BaseStrategy):
             return self._encode_subobject(o)
+        if isinstance(o, Duration):
+            return None
```

Comment:
Can you please explain this change? Don't we want to use `.format()` or something similar?

Reply:
I explicitly did it this way in order to match the previous behavior. Up until now, all objects that were not able to be serialized were omitted. I changed this behavior to raise an exception in these cases, and the exception was not raised for any other objects apart from a duration object. Therefore to mimic the old behavior I omitted it. If you want though, I can try to serialize this too.
WRT the serialization itself, the problem is that format() cannot be used on Duration objects, therefore I will do a Duration.seconds instead.

Reply:
In this case, I agree, leave the current behaviour. Thanks!